docs/podman-pod-create.1.md

```diff
@@ -0,0 +1,71 @@
+% podman-pod-create "1"
+
+## NAME
+podman\-pod\-create - Create a new pod
+
+## SYNOPSIS
+**podman pod create** [*options* [...]] IMAGE [COMMAND] [ARG...]
```

Comment:
This is the old approach.  Since #1027, this should look like:

```markdown
**podman pod create** [*options*] *image* [*command* [*arg* ...]]
```

or similar, depending on what arguments you actually have `pod create` accept.